keycloak/keycloak_entrypoint.sh

```diff
@@ -28,7 +28,7 @@ fi
 
 # Start Keycloak in the background
 echo "Starting Keycloak"
-/opt/keycloak/bin/kc.sh start-dev --features=preview --import-realm -Dkeycloak.profile.feature.scripts=enabled -Dkeycloak.migration.strategy=OVERWRITE_EXISTIN &
+/opt/keycloak/bin/kc.sh start-dev --log-level=DEBUG --features=preview --import-realm -Dkeycloak.profile.feature.scripts=enabled -Dkeycloak.migration.strategy=OVERWRITE_EXISTIN &
```

Comment:
Am I right that this docker-compose will be used for debug mostly? Debug logs helped me a lot.

Reply:
I guess we will need smth else for production. There is whole issue around Keycloak with TLS and certificate. Mainly this `start-dev` because `kc.sh start` won't work without TLS.